packages/app-admin/src/plugins/Menu/community.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { AdminMenuCommunityPlugin } from "@webiny/app-admin/types";
+import { ListItem, ListItemGraphic } from "@webiny/ui/List";
+import { Icon } from "@webiny/ui/Icon";
+import { ReactComponent as CommunityIcon } from "@webiny/app-admin/assets/icons/icon-community.svg";
+
+import { i18n } from "@webiny/app/i18n";
+const t = i18n.ns("app-admin/navigation");
+
+const plugin: AdminMenuCommunityPlugin = {
+    type: "admin-menu-community",
+    name: "admin-menu-community",
+    render() {
+        return (
+            <>
+                <a
+                    href="https://community.webiny.com/"
```

Comment:
@BhuwanChandra also please change `Community` with `Slack` as mentioned in the [original issue](https://github.com/webiny/webiny-js/issues/1259#issuecomment-697247618).

This means changing the link, text, and icon of the menu item.